index.js

```diff
@@ -9,7 +9,9 @@ const corePlugins = [
   signoffPlugin
 ];
 
-const settings = {};
+const settings = {
+  maxPerPage: 10,
+};
```

Comment:
Yes but that makes me think: if we don't specify a `limit` we're gonna skip the feature and paginate by server defaults, which may be high and inconvenient to use.

Is it okay of should I create a distinct `historyMaxPerPage` setting, so we can have something usable by default in the admin?